std-2/5_AttendanceRegister/AttendanceRegister.sol

```diff
@@ -15,7 +15,7 @@ contract AttendanceRegister {
 
     event Added(string name, uint class, uint time);
 
-    mapping(uint => Student) public register; // roll number => student details
+    mapping(uint => Student) public registered; // roll number => student details
```

Comment:
this should be update in the complete contract wherever used.

Reply:
Fixed